composer.json

```diff
@@ -22,7 +22,7 @@
         "ext-dom": "*",
         "monolog/monolog": "~1.7||~2.0",
         "phpunit/php-file-iterator": "~2.0||^3.0",
-        "symfony/console": "~3.4||~4.0||~5.0"
+        "symfony/console": "~3.4||~4.0||~5.0||~6.0"
```

Comment:
This is not directly related, but anyway I think that being compatible with Symfony 6 can be nice